app/hotels/src/navigation/allHotels/SearchResultsScreen.js

```diff
@@ -44,20 +46,15 @@ type Props = {|
 const noop = () => {};
 
 const SearchResultsScreen = (props: Props) => {
-  React.useEffect(() => {
-    props.navigation.setParams({
-      toggleMap: props.setResultType,
-      setCheckinDate: props.setCheckinDate,
-      setCheckoutDate: props.setCheckoutDate,
-    });
-  }, []);
```

Comment:
So with the new implementation, all changes to the values are going to trigger a new `useEffect`, when the previous one set `toggleMap`, `setCheckinDate` and `setCheckoutDate` at the initial render only, and not in subsequent re-renders.
Is that a "regression" in some sense?

Reply:
After doing some research on the eslint-rule, and specifically this [comment](https://github.com/facebook/react/issues/14920#issuecomment-470863122) I think we should have all props in the array. 

None of this props will change, but in theory they could.

Reply:
Okay, makes sense! Thanks for sharing your research!